.config/cargo_spellcheck.dic

```diff
@@ -0,0 +1,75 @@
+80
+
+WebAssembly
+Wasm
+Polkadot
+validator
+LLVM
+JIT
+rustup
+miri
+clippy
+hunspell
+ERC
+fuzzer
+1A
+2A
+1B
+2B
```

Comment:
Things like these should not be part of this file and rather be replace with some proper respelling or reformatting.